metl/main.py

```diff
@@ -106,34 +107,37 @@ def _get_data_encoding(hparams):
     return encoding
 
 
-def load_model_and_data_encoder(state_dict, hparams):
+def load_model_and_data_encoder(state_dict, hparams, strict, raw, indexing):
```

Comment:
This approach looks clean to me. If we all agree to lock in this design, having comments somewhere in this file about why we set strict, raw, and indexing to certain defaults in various functions will be useful.

Reply:
I added documentation to the get functions, the load functions and to ModelEncoder.validate_pdb.